lib/minitest/reporters/ansi.rb

```diff
@@ -2,7 +2,13 @@ module Minitest
   module Reporters
     module ANSI
       module Code
-        if ($stdout.tty?)
+
+        def self.color?
+          color_terminal = ENV['TERM'].to_s.downcase.index("color")
```

Comment:
Because it's faster :D   
No reason really, except that index just came to mind.